app/controllers/organisations_controller.rb

```diff
@@ -1,6 +1,8 @@
 class OrganisationsController < PublicFacingController
   include CacheControlHelper
 
+  CONTENT_NOT_FOUND = Class.new
```

Comment:
Ah I didn't think about the memoization of the rescue block return value. Makes more sense now.
Perhaps for the sake of clarity a custom class with a descriptive name could be used?
`MissingContentItem`, `NonExistentContentItem` or  something similar (admittedly neither of these suggestions really stands out).